integration/sql/iface-java/src/java/io/openlineage/sql/DbTableMeta.java

```diff
@@ -0,0 +1,69 @@
+/*
+/* Copyright 2018-2022 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.sql;
+
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+public class DbTableMeta {
+    private final String database;
+    private final String schema;
+    private final String name;
+    
+    public DbTableMeta(String database, String schema, String name) {
+        this.database = database;
+        this.schema = schema;
+        this.name = name;
+    }
+
+    public String database() {
+        return database;
+    }
+
+    public String schema() {
+        return schema;
+    }
+
+    public String name() {
+        return name;
+    }
+
+    public String qualifiedName() {
+        return String.format(
+            "%s%s%s",
+            database != null ? database + "." : "",
+            schema != null ? schema + "." : "",
+            name
+        );
+    }
+
+    @Override
+    public String toString() {
+        return qualifiedName();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o == this) {
+            return true;
+        }
+
+        if (!(o instanceof DbTableMeta)) {
+            return false;
+        }
+
+        DbTableMeta other = (DbTableMeta)o;
+        return qualifiedName().equals(other.qualifiedName());
```

Comment:
Qualified name has no dots or other separators.
So `equals` can be true even when being not equal.

Reply:
Is there database where `database`, `schema` and `table` are separated by anything else than a dot? If yes that's a problem, but I don't know that it exists.

Reply:
you're right, I didn't notice how dots are filled within `qualifiedName` method,